arch/sim/Kconfig

```diff
@@ -23,6 +23,10 @@ config HOST_ARM
 	bool "arm"
 	select ARCH_HAVE_STACKCHECK
 
+config HOST_ARM64
```

Comment:
since the skip of objcopy is specific to clang toolchain, it's better to fix the visibility issue to a standalone patch